addons/web/controllers/main.py

```diff
@@ -1030,6 +1030,22 @@ def test_mobile_suite(self, mod=None, **kwargs):
     def benchmarks(self, mod=None, **kwargs):
         return request.render('web.benchmark_suite')
 
+    @http.route('/web/bundle/<string:bundle_name>', type='http', auth="user", methods=["GET"])
+    def bundle(self, bundle_name):
+        """
+        Request the definition of a bundle, including its javascript and css bundled assets
+        One might think that making this controller type=json is a good idea, but in fact odoo cannot receive Get
+        requests for routes with type=json that has no body, and the browser's fetch API cannot generate a Get request
+        that has a body.
```

Comment:
This is not specific to the fetch API: the body in GET requests has no semantic meaning defined by the HTTP spec and relying on it can have unexpected results when you don't control the entire HTTP stack (eg if you are behind a CDN). Since this route no longer takes any parameter I think it's fine to leave it as a GET request and explicitly set the `Content-type` header as you did, it saves us from performing a well-formed RPC for something that doesn't need it.